qemu/tests/cfg/multi_nics_verify.cfg

```diff
@@ -1,10 +1,12 @@
 - multi_nics_verify: guest_test.arp_set
     virt_test_type = qemu
     type = multi_nics_verify
+    image_snapshot = yes
     no RHEL.3.9
     start_vm = no
     login_timeout = 3600
     flexible_nic_index = yes
+    check_guest_ver = "awk '/PRETTY_NAME/ {print}' /etc/os-release | awk '{print $5}' | awk -F '.' '{print $1}'"
```

Comment:
I consider this problem before I sent this PR, due to the steps to create the two are somewhat different， so it can not be limited by `RHEL.x `. It is more suitable for limiting the cases where the steps are the same but the commands are different. Please correct me if I'm wrong.

Reply:
But this cmd is checking the release X version, same as RHEL.X. My aim is like:
```
RHEL.7, RHEL.8:
    dhcp_manager = dhclient
dhcp_manager = nmcli
```

So we can check the value of it and do different steps.

Reply:
Hi @PaulYuuu I think there are some gap between us, the reason is that we have discussed problem about dhcp in the past. But this case is not related it. The current scenario is add 27 nics to guest, then boot up and create config/key  file, then reboot guest and re-check all nics can got ip address. There will be no significant improvement for test steps that we check the release version in the cfg .

Reply:
OK, but the cmd is really to check the release version like RHEL.x does. Leave aside dhcp, the above suggestion can be:
```
RHEL.7, RHEL.8:
    network_manager = network-sciprts
network_manager = nmcli
```

If using non-RHEL guest, the `PRETTY_NAME` is much diff.

Reply:
Ok, got it, let me try to re-change it.

Reply:
cfg:
```
network_manager = yes
RHEL.7, RHEL.8:
    network_manager = no
```
python file:
 ```
network_manager = params.get("network_manager")
if network_manager == "yes":
    for ifname in ifname_list:
        eth_keyfile_path = keyfile_path % ifname
         cmd = "nmcli --offline connection add type ethernet con-name %s ifname %s" \
                      " ipv4.method auto > %s" % (ifname, ifname, eth_keyfile_path)
         s, o = session.cmd_status_output(cmd)
         if s != 0:
             err_msg = "Failed to create ether keyfile: %s\nReason is: %s"
             test.error(err_msg % (eth_keyfile_path, o))
         session.cmd("chown root:root /etc/NetworkManager/system-connections/*.nmconnection")
         session.cmd("chmod 600 /etc/NetworkManager/system-connections/*.nmconnection")
         session.cmd("nmcli connection reload")
else:
    for ifname in ifname_list:
        eth_config_path = ifcfg_path % ifname
        eth_config = "DEVICE=%s\\nBOOTPROTO=dhcp\\nONBOOT=yes" % ifname
        cmd = "echo -e '%s' > %s" % (eth_config, eth_config_path)
        s, o = session.cmd_status_output(cmd)
            if s != 0:
                err_msg = "Failed to create ether config file: %s\nReason is: %s"
                test.error(err_msg % (eth_config_path, o))
```

Hi @PaulYuuu How about the above usage ? Please feel free to add your comment. Many thanks.

Reply:
It's okay for me, but don't forget to use get_boolean.